spec/services/bulk_sql_delete_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
+
+describe BulkSqlDelete, type: :service do
+  let(:sql) do
+    <<-SQL
+      DELETE FROM notifications
+      WHERE notifications.id IN (
+        SELECT notifications.id
+        FROM notifications
+        WHERE created_at < '#{Time.zone.now}'
+        LIMIT 1
+      )
+    SQL
+  end
```

Comment:
I wonder if it'd be easier to have this as a constant not to have to write it twice but we can live with this for now